ouroboros-consensus/docs/Onboarding.md

```diff
@@ -37,14 +37,17 @@ nix-env (Nix) 2.3.10
 > If configured correctly, your fresh build (that means build from completely new
 > machine) should take ~15 minutes
 
-Open your `~/.config/nix/nix.conf` and add/modify following entries
+If you are using a single-user installation of Nix, open your
+`~/.config/nix/nix.conf` and add/modify following entries
 
 ```
 substituters = https://cache.nixos.org/ https://hydra.iohk.io/
 trusted-substituters = https://cache.nixos.org/ https://hydra.iohk.io/
 trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=
 ```
 
+If instead you are using a multi-user installation, add/modify said entries in `/etc/nix/nix.conf`.
```

Comment:
Would it make sense to refer to the cache misses that one might run into if this is not correctly set?

Reply:
I think so. Will add it later today :+1:

Reply:
Done :ok_hand: